contrib/guix/libexec/build.sh

```diff
@@ -61,9 +61,10 @@ case "$HOST" in
         # When targeting darwin, some native tools built by depends require
         # native packages not incorporated in depends
         #
-        # libcap       required by native_cdrkit/wodim
-        # zlib, bzip2  required by native_cdrkit/genisoimage
-        for native_pkg in libcap zlib bzip2; do
+        # zlib         required by native_libdmg-hfsplus
+        #
+        # shellcheck disable=SC2043
```

Comment:
The change looks correct, but can't we just remove the loop, rather than adding more shellcheck cruft to disable a warning about what is a pointless loop? You could then also make the comment above zlib specific. i.e: "When targeting darwin, a native zlib is required  by libdmg-hfsplus", and just remove the listing.

Reply:
My thought was if another dependency appears in the future this approach allows to just add a package name into the loop.

But your suggestion is more neat. Going to implement it.

Reply:
Thanks! [Updated](https://github.com/bitcoin/bitcoin/pull/21337#issuecomment-789484190).

Reply:
> My thought was if another dependency appears in the future this approach allows to just add a package name into the loop.

If we need to add more packages (I'd think unlikely at this stage), we can just re-add the loop.